### ISSUE TITLE
position_controllers/GripperActionController is in gripper_controllers package?

### ISSUE THREAD

**Issue by tylerjw** [ros-controls/ros2_controllers#223]
This confused me because I'm trying to track down dependencies.  There is a `position_controllers` package but it does not contain the `GripperActionController`, instead that is inside the `gripper_controllers` package but if you look at the XML description of that plugin it is `position_controllers/GripperActionController`.  Would it make sense to change the name of the plugin to `gripper_controllers/GripperActionController` to help avoid confusing others?

**Comment by destogl** [comment]
I agree with you. We missed this when porting to ROS2. @bmagyar, @Karsten1987 what do you think?

**Comment by bmagyar** [comment]
Not a problem for me
